src/NorthwindWebsite/Core/ApplicationSettings/EmailSender.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NorthwindWebsite.Core.ApplicationSettings
+{
+    public class EmailSenderConfigs
+    {
+        public string SendersEmail { get; set; }
```

Comment:
Probably, it's also a good idea to add `Required` attribute to make sure you will not pass falsy values to your message client. Same for `AuthMessageSenderOptions`